extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/util/PropertyUtils.java

```diff
@@ -41,8 +46,13 @@ public static State getState(ChannelUID channelUID, AstroChannelConfig config, O
         if (value == null) {
             return UnDefType.UNDEF;
         } else if (value instanceof Calendar) {
-            Calendar cal = (Calendar) value;
-            return new DateTimeType(DateTimeUtils.applyConfig(cal, config));
```

Comment:
The call to `DateTimeUtils.applyConfig(cal, config)` is not reflected in your changes. The configured offset is not taken into account anymore.